rust/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-04-22
+nightly-2020-08-24
```

Comment:
I understand this upgrade was helpful during development, but I suggest we don't upgrade upon final merge.